src/EditorFeatures/Test/Diagnostics/GenerateType/TestGenerateTypeOptionsService.cs

```diff
@@ -40,7 +40,7 @@ internal class TestGenerateTypeOptionsService : IGenerateTypeOptionsService
             // Storing the actual values
             ClassName = className;
             GenerateTypeDialogOptions = generateTypeDialogOptions;
-
+            var defaultNamespace = projectManagementService.GetDefaultNamespace(Project, Project?.Solution.Workspace);
```

Comment:
The tests all set the default namespace through the [TestProjectManagementService](http://source.roslyn.io/#Roslyn.Services.Editor.UnitTests/Diagnostics/GenerateType/TestProjectManagementService.cs,11).  We could re-factor the test infrastructure to not require this type.  I don't know what benefit we get from explicitly setting the namespace in   TestGenerateTypeOptionsService vs. TestProjectManagementService